bundle/language_en.properties

```
@@ -785,6 +785,8 @@ angal.medicals.thismonth
 angal.medicals.today                                                                                   = Today
 angal.medicals.type                                                                                    = Type
 angal.medicals.withlot                                                                                 = With lots
+angal.medicalstock.allcharges                                                                          = All charges
+angal.medicalstock.alldischarges                                                                       = All discharges
```

```suggestion
angal.medicalstock.alldischarges.txt                                                                   = All discharges
```
change references too